src/main/java/com/github/mob41/blapi/pkt/CmdPacket.java

```diff
@@ -108,9 +108,19 @@ public CmdPacket(Mac targetMac, int count,
 
         log.debug("Running checksum for headers");
 
+		/* The program is ported from python.
+		 * Byte in python in the range is 0 to 256, but in java, it is in the range of -128 to 127.
+		 * Here is a judgment to ensure that payload[i] is a positive number and checksum is aways growing.         *         * 
+		 *  
+		 * The same problem exists in data[0x20] and data[0x21], but it does not affect the execution of the program.
+		 */
         int checksum = 0xbeaf;
         for (int i = 0; i < payload.length; i++){
-            checksum += payload[i];
+        	if(payload[i]>=0){

```

Comment:
Also, there is indentation problems in the lines you modified. Please convert all tabs to spaces (one tab is four spaces) before merging. If you are using Eclipse IDE, see [this](https://stackoverflow.com/questions/407929/how-do-i-change-eclipse-to-use-spaces-instead-of-tabs) and change settings and do ```(Right click project) -> Source -> Format```